tests/FTPServiceTest.php

```diff
@@ -80,7 +80,7 @@ public function testDelete()
         $result = $this->ftp_service->delete($file_name);
 
         $this->assertTrue($result);
-        $this->setExpectedException(\Bow\Storage\Exception\ResourceException::class);
+        $this->expectException(\Bow\Storage\Exception\ResourceException::class);
```

Comment:
To upgrade the PHPUnit version to `8`, it should change the exception throwing assertion.